src/diffusers/loaders/lora_conversion_utils.py

```diff
@@ -798,6 +989,23 @@ def handle_qkv(sds_sd, ait_sd, sds_key, ait_keys, dims=None):
     return new_state_dict
 
 
+def _custom_replace(key: str, substrings: List[str]) -> str:
+    pattern = "(" + "|".join(re.escape(sub) for sub in substrings) + ")"
```

Comment:
maybe a short comment/docstring with an example of what this function does would help